src/datepicker/datepicker-month-view.ts

```diff
@@ -0,0 +1,54 @@
+import {Component, Input, TemplateRef, Output, EventEmitter} from '@angular/core';
+import {MonthViewModel, DayViewModel, DayTemplateContext} from './datepicker-view-model';
+
+@Component({
+  selector: 'tbody[ngb-datepicker-month-view]',
+  styles: [`
+    .weekday {
+      text-align: center;
+      font-weight: bold;
+      width: 30px;
+      height: 30px;
+    }
+    .weeknumber {
+      text-align: center;
+      font-size: 12px;
+      color: #444;
```

Comment:
This would be no-go as it would kill custom themes. Sizes expressed in pixels will also kill zoom in / out no? 

More generally we should try to limit custom styling here as users won't have any way of overriding / customizing it. As discussed one way would be to use buttons. 

This part needs more investigation / work.